### ISSUE TITLE
installation error in v.1.7.1.

### ISSUE THREAD

**Issue by mzch** [nokonoko/Uguu#84]
When doing `make`, the following error happens:
```
mkdir -p /srv/uguu/Uguu/build /srv/uguu/Uguu/build/img /srv/uguu/Uguu/build/html /srv/uguu/Uguu/build/html/min /srv/uguu/Uguu/build/html/unmin /srv/uguu/Uguu/build/js /srv/uguu/Uguu/build/css /srv/uguu/Uguu/build/php /srv/uguu/Uguu/build/php/Classes  /srv/uguu/Uguu/build/public
"npm" install

up to date, audited 500 packages in 720ms

82 packages are looking for funding
  run `npm fund` for details

22 vulnerabilities (12 moderate, 10 high)

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
"node_modules/ejs/bin/cli.js" -f /srv/uguu/Uguu/"src/config.json" /srv/uguu/Uguu/src/templates/index.ejs -o /srv/uguu/Uguu/build/html/unmin/index.html;  "node_modules/ejs/bin/cli.js" -f /srv/uguu/Uguu/"src/config.json" /srv/uguu/Uguu/src/templates/faq.ejs -o /srv/uguu/Uguu/build/html/unmin/faq.html;  "node_modules/ejs/bin/cli.js" -f /srv/uguu/Uguu/"src/config.json" /srv/uguu/Uguu/src/templates/tools.ejs -o /srv/uguu/Uguu/build/html/unmin/tools.html;
/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:361
  throw err;
  ^

Error: ejs:1
 >> 1| <%- include('./partials/header.ejs'); -%>
    2| <div class="container">
    3|     <div class="jumbotron">
    4|         <%- include('./partials/lead.ejs'); -%>

Could not find the include file "./partials/header.ejs"
    at getIncludePath (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:185:13)
    at includeFile (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:311:19)
    at include (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:701:16)
    at eval (eval at compile (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:673:12), <anonymous>:10:17)
    at anonymous (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:703:17)
    at Object.exports.render (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:425:37)
    at Immediate.run (/srv/uguu/Uguu/node_modules/ejs/bin/cli.js:201:20)
    at processImmediate (node:internal/timers:466:21) {
  path: ''
}
/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:361
  throw err;
  ^

Error: ejs:1
 >> 1| <%- include('./partials/header.ejs'); -%>
    2| <div class="container">
    3|     <article>
    4|         <div class="jumbotron">

Could not find the include file "./partials/header.ejs"
    at getIncludePath (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:185:13)
    at includeFile (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:311:19)
    at include (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:701:16)
    at eval (eval at compile (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:673:12), <anonymous>:10:17)
    at anonymous (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:703:17)
    at Object.exports.render (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:425:37)
    at Immediate.run (/srv/uguu/Uguu/node_modules/ejs/bin/cli.js:201:20)
    at processImmediate (node:internal/timers:466:21) {
  path: ''
}
/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:361
  throw err;
  ^

Error: ejs:1
 >> 1| <%- include('./partials/header.ejs'); -%>
    2| <div class="container">
    3|     <div class="jumbotron">
    4|         <h1><%= siteName %> Tools</h1>

Could not find the include file "./partials/header.ejs"
    at getIncludePath (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:185:13)
    at includeFile (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:311:19)
    at include (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:701:16)
    at eval (eval at compile (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:673:12), <anonymous>:10:17)
    at anonymous (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:703:17)
    at Object.exports.render (/srv/uguu/Uguu/node_modules/ejs/lib/ejs.js:425:37)
    at Immediate.run (/srv/uguu/Uguu/node_modules/ejs/bin/cli.js:201:20)
    at processImmediate (node:internal/timers:466:21) {
  path: ''
}
make: *** [Makefile:29: ejs] Error 1
```

After patching below, `make` has done correctly:
```diff
--- templates/index.ejs.bak     2023-04-27 03:17:46.085446239 +0900
+++ templates/index.ejs 2023-04-27 03:21:58.756749138 +0900
@@ -1,15 +1,15 @@
-<%- include('./partials/header.ejs'); -%>
+<%- include('./src/templates/partials/header.ejs'); -%>
 <div class="container">
     <div class="jumbotron">
-        <%- include('./partials/lead.ejs'); -%>
-        <%- include('./partials/upload_form.ejs'); -%>
+        <%- include('./src/templates/partials/lead.ejs'); -%>
+        <%- include('./src/templates/partials/upload_form.ejs'); -%>
     </div>
 
     <% if (malwareBanner) { %>
-        <%- include('./partials/malware_banner.ejs'); -%>
+        <%- include('./src/templates/partials/malware_banner.ejs'); -%>
     <% } %>
 
     <% if (donationBanner) { %>
-        <%- include('./partials/donation_banner.ejs'); -%>
+        <%- include('./src/templates/partials/donation_banner.ejs'); -%>
     <% } %>
-<%- include('./partials/footer.ejs'); -%>
+<%- include('./src/templates/partials/footer.ejs'); -%>
```
```diff
-- templates/faq.ejs.bak       2023-04-27 03:18:01.989391103 +0900
+++ templates/faq.ejs   2023-04-27 03:20:11.501002239 +0900
@@ -1,4 +1,4 @@
-<%- include('./partials/header.ejs'); -%>
+<%- include('./src/templates/partials/header.ejs'); -%>
 <div class="container">
     <article>
         <div class="jumbotron">
@@ -61,4 +61,4 @@
 
         </div>
     </article>
-<%- include('./partials/footer.ejs'); -%>
+<%- include('./src/templates/partials/footer.ejs'); -%>
```
```diff
--- templates/tools.ejs.bak     2023-04-27 03:18:18.193336712 +0900
+++ templates/tools.ejs 2023-04-27 03:20:33.076946792 +0900
@@ -1,4 +1,4 @@
-<%- include('./partials/header.ejs'); -%>
+<%- include('./src/templates/partials/header.ejs'); -%>
 <div class="container">
     <div class="jumbotron">
         <h1><%= siteName %> Tools</h1>
@@ -56,4 +56,4 @@
             </dl>
         </section>
     </div>
-<%- include('./partials/footer.ejs'); -%>
+<%- include('./src/templates/partials/footer.ejs'); -%>
```

**Comment by nokonoko** [comment]
Works fine for me on MacOS and Linux (Debian), both running ZSH shell.

What OS and u running and what shell?

**Comment by mzch** [comment]
I'm running Debian `11.6` and using `bash`.

**Comment by nokonoko** [comment]
Cannot re-produce, work fine on Debian, using bash on 1.7.4.

Are you on AMD64 or ARM64?

I'd say try again using the newest documentation and code.

**Comment by nokonoko** [comment]
I managed to re-produce this error on a newer version of Node and has been fixed with the release of V1.7.5.
